aspnetcore/security/authentication/scaffold-identity/consoleAddUser/Data/SeedData.cs

```diff
@@ -1,71 +1,54 @@
 ﻿using Microsoft.AspNetCore.Identity;
-using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
 using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using WebApplication1.Data;
 
 namespace ContactManager.Data
 {
     public static class SeedData
     {
         #region snippet
 
-        public static async Task Initialize(IServiceProvider serviceProvider, string userList)
+        public static async Task Initialize(IServiceProvider serviceProvider, List<string> userList)
```

Comment:
Format this so we don't get a horizontal scroll bar

![image](https://user-images.githubusercontent.com/3605364/67527299-65f7ca80-f652-11e9-9a82-ec6b4299f493.png)